src/Pact/Eval.hs

```diff
@@ -283,6 +283,7 @@ reduce (TObject ps t i) =
 reduce (TBinding ps bod c i) = case c of
   BindLet -> reduceLet ps bod i
   BindSchema _ -> evalError i "Unexpected schema binding"
+  BindInlinedCallArgs -> evalError i "Unexpected inlined call args binding"
```

Comment:
Woah nellie. Sorry I never meant this to bleed into `Term`. Can you use a new enum specific to `AST` and add this type? It's meaningless in pact at large.

Reply:
OK -- sure. Sounds good